fvm/transactionVerifier.go

```diff
@@ -240,6 +285,18 @@ func (v *TransactionVerifier) verifyTransaction(
 		if addr == tx.Payer {
 			continue
 		}
+
+		if serviceAccountOveride && addr != serviceAddress {
+			exists, err := accounts.Exists(addr)
+			if err != nil {
+				return err
+			}
+			if !exists {
+				return errors.NewInvalidProposalSignatureError(tx.ProposalKey, fmt.Errorf("account %s does not exist", addr))
+			}
+			continue
```

Comment:
should we be checking the service accounts key weight here?
```suggestion
			if !v.hasSufficientKeyWeight(payloadWeights, serviceAddress, keyWeightThreshold) {
				return errors.NewAccountAuthorizationErrorf(
					serviceAddress,
					"authorizer account does not have sufficient signatures (%d < %d)",
					payloadWeights[serviceAddress],
					keyWeightThreshold)
			}
			continue
```